File: 2.Traverse/Python/6.reverse_traversal.py
def reverse_traversal(n, arr):
  matrix = []
  for i in range(n-1,-1,-1):
    matrix.append(arr[i])
  
  return matrix

n=5
arr = [1, 3, 2, 4, 5]
res =  reverse_traversal(n, arr)
print(*res)

File: 2.Traverse/Javascript/6.reverse_traversal.js
function reverseTraversal(n, arr) {
    let matrix = [];
    for (let i = n - 1; i >= 0; i--) {
        matrix.push(arr[i]);
    }
    return matrix;
}

let n = 5;
let arr = [1, 3, 2, 4, 5];
let res = reverseTraversal(n, arr);
console.log(...res);
